.circleci/config.yml

```diff
@@ -797,7 +799,7 @@ jobs:
           determined-common: true
           determined-cli: true
           determined: true
-          extras-requires: "tensorflow==1.14 torch==1.4"
+          extras-requires: "tensorflow==2.4 torch==1.8"
```

Comment:
wdy think about moving torch1.8 into the requirements.txt file for this module?

Reply:
Discussed offline: let's handle that separately and just keep the delta here as small as possible.